addons/website_event_sale/tests/test_frontend_buy_tickets.py

```diff
@@ -24,28 +26,13 @@ def setUp(self):
             'is_published': True,
         })
 
-        self.event_2 = self.env['event.event'].create({
-            'name': 'Conference for Architects TEST',
```

Comment:
Seems the tour cared about the name of this event :grimacing: 
I'm sorry that tours exist at all